README.md

```diff
@@ -584,6 +584,7 @@ to remove the comments holding dump action tracing.
 1. Run `make diff-releases` to check the changed BOSH properties; see
    the next section for details.
 1. Run `tooling/bin/check-uaa-clients`
+1. Bump [CloudFoundry Acceptance Tests(CATs)](https://github.com/cloudfoundry/cf-acceptance-tests) submodule to match new `CF_VERSION`, before testing.
```

Comment:
I know you are just following the schema used above, but Markdown is supposed to be readable as plain text, and using the same number for each bullet point kind of defeats that purpose.

Reply:
Sure, trivial to fix.

Reply:
Done with commit e7f3d54339f2021d8795054b2f97886a71f100b6